vms/volunteer/views.py

```diff
@@ -54,10 +55,10 @@ def delete_resume(request, volunteer_id):
                 return HttpResponseRedirect(reverse('volunteer:profile', args=(volunteer_id,)))
             except:
                 raise Http404
+             
     else:
         return HttpResponse(status=403)
-
-'''
+'''         

```

Comment:
Can you please run flake8 on your changes?

Reply:
![flake](https://user-images.githubusercontent.com/22135941/31664117-d2d49a9c-b361-11e7-8cd9-16904d0faebb.png)

I get this on running flake8 , shall I go do all these changes?

Reply:
Yes.

Reply:
Please let me know if you are stuck on something for long, we can see it together. Are you still working on this?

Reply:
Yes , I'm doing the flake8 changes .

Reply:
Is this pr still valid? Can I push the commits with Flake8 chnages?

Reply:
If this is the final one, you need to squash your commits.

Reply:
I created a cleaner pr with the suggested changes here #513. So closing this one.